composer.json

```diff
@@ -7,16 +7,18 @@
     "require": {
         "php": "^7.1.3",
         "backup-manager/laravel": "^1.2",
-        "laracasts/presenter": "^0.2.1",
         "fideloper/proxy": "^4.0",
+        "laracasts/presenter": "^0.2.1",
         "laravel/framework": "5.7.*",
         "luthfi/formfield": "^1.0",
         "riskihajar/terbilang": "^1.2",
-        "spatie/laravel-fractal": "^5.0"
+        "spatie/laravel-fractal": "<5.5",
+        "spatie/laravel-medialibrary": "^7.0.0"
```

Comment:
I do not want to use this Package for file upload. please use regular file upload and create another PR for job attachment feature.